app/src/main/java/com/nextcloud/talk/activities/FullScreenImageActivity.kt

```diff
@@ -188,6 +228,51 @@ class FullScreenImageActivity : AppCompatActivity() {
         }
     }
 
+    @SuppressLint("LongLogTag")
+    private fun saveImageToStorage(
+        fileName: String,
+        funToCallWhenDownloadSuccessful: (() -> Unit)
```

Comment:
the second parameter (funToCallWhenDownloadSuccessful) is not used so it could also just be deleted.
So just call the method with saveImageToStorage(fileName) and delete the observer.